miner/worker.go

```diff
@@ -425,21 +424,6 @@ func (miner *Miner) commitTransactions(env *environment, plainTxs, blobTxs *tran
 		if !env.txFitsSize(tx) {
 			break
 		}
-
-		// Make sure all transactions after osaka have cell proofs
-		if isOsaka {
```

Comment:
The conversion is unnecessary and useless.

If only affects the referenced transaction object, while the one in the blobpool/datadir won't be changed.

What really matters if the ones in the blobpool, for serving GetBlob APIs at the engine api.